db/hints/manager.cc

```diff
@@ -904,7 +905,7 @@ future<> manager::end_point_hints_manager::sender::send_one_hint(lw_shared_ptr<s
                 return make_exception_future<>(std::move(eptr));
             }
             return make_ready_future<>();
-        }).then_wrapped([this, units = std::move(units), rp, ctx_ptr] (future<>&& f) {
+        }).then_wrapped([this, units = std::move(units), rp, ctx_ptr, h = std::move(h)] (future<>&& f) {
```

Comment:
It sounds like a fix we would want to backport, please open a github ticket. \cc @vladzcloudius

Reply:
https://github.com/scylladb/scylladb/issues/15110

Reply:
done ([it shouldn't contain colon btw](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))

Reply:
I still see the old commit message without the Fixes

Reply:
Have no idea what happened to it, I clearly remember I updated the commit message